tests/cisco_ios/show_ip_interface_brief/cisco_ios_show_ip_interface_brief.raw

```diff
@@ -4,4 +4,5 @@ Ethernet0/0.11             10.0.1.38       YES NVRAM  up                    up
 Ethernet0/1                1.1.1.1         YES NVRAM  up                    up
 Ethernet0/2                unassigned      YES NVRAM  administratively down down
 Ethernet0/3                unassigned      YES NVRAM  administratively down down
+Ethernet0/4                unassigned      YES NVRAM  deleted               down
```

Comment:
@averhaven
This is more about a sub-interface that is deleted and not a physical interface, correct?